src/EnergyPlus/DemandManager.cc

```diff
@@ -1079,6 +1092,116 @@ namespace DemandManager {
 
 			} // MgrNum
 
+			// Get input for DemandManager:Ventilation
+			StartIndex = EndIndex + 1;
+			EndIndex += NumDemandMgrVentilation;
+
+			CurrentModuleObject = "DemandManager:Ventilation";
+
+			for (MgrNum = StartIndex; MgrNum <= EndIndex; ++MgrNum) {
+
+				GetObjectItem(CurrentModuleObject, MgrNum - StartIndex + 1, AlphArray, NumAlphas, NumArray, NumNums, IOStat, _, lAlphaFieldBlanks, cAlphaFieldNames, cNumericFieldNames);
+
+				IsNotOK = false;
+				IsBlank = false;
+				VerifyName(AlphArray(1), DemandMgr.Name(), MgrNum - StartIndex, IsNotOK, IsBlank, CurrentModuleObject + " Name");
+				if (IsNotOK) {
+					ErrorsFound = true;
+					if (IsBlank) AlphArray(1) = "xxxxx";
+				}
+				DemandMgr(MgrNum).Name = AlphArray(1);
+
+				DemandMgr(MgrNum).Type = ManagerTypeVentilation;
+
+				if (!lAlphaFieldBlanks(2)) {
+					DemandMgr(MgrNum).AvailSchedule = GetScheduleIndex(AlphArray(2));
+
+					if (DemandMgr(MgrNum).AvailSchedule == 0) {
+						ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid " + cAlphaFieldNames(2) + "=\"" + AlphArray(2) + "\" not found.");
+						ErrorsFound = true;
+					}
+				}
+				else {
+					DemandMgr(MgrNum).AvailSchedule = ScheduleAlwaysOn;
+				}
+
+				// Validate Limiting Control
+				{ auto const SELECT_CASE_var(AlphArray(3));
+				if (SELECT_CASE_var == "OFF") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitOff;
+
+				}
+				else if (SELECT_CASE_var == "FIXEDRATE") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitFixed;
+
+				}
+				else if (SELECT_CASE_var == "REDUCTIONRATIO") {
+					DemandMgr(MgrNum).LimitControl = ManagerLimitReductionRatio;
+
+				}
+				else {
+					ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid value" + cAlphaFieldNames(3) + "=\"" + AlphArray(3) + "\".");
+					ShowContinueError("...value must be one of Off, Fixed, or Variable.");
+					ErrorsFound = true;
+				}}
+
+				DemandMgr(MgrNum).LimitDuration = NumArray(1);
+
+				DemandMgr(MgrNum).FixedRate = NumArray(2);
+				DemandMgr(MgrNum).ReductionRatio = NumArray(3);
+
+				DemandMgr(MgrNum).LowerLimit = NumArray(4);
+
+				// Validate Selection Control
+				{ auto const SELECT_CASE_var(AlphArray(4));
+				if (SELECT_CASE_var == "ALL") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionAll;
+
+				}
+				else if (SELECT_CASE_var == "ROTATEONE") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionOne;
+
+				}
+				else if (SELECT_CASE_var == "ROTATEMANY") {
+					DemandMgr(MgrNum).SelectionControl = ManagerSelectionMany;
+
+				}
+				else {
+					ShowSevereError(CurrentModuleObject + "=\"" + cAlphaArgs(1) + "\" invalid value" + cAlphaFieldNames(4) + "=\"" + AlphArray(4) + "\".");
+					ShowContinueError("...value must be one of All, RotateOne, or RotateMany.");
```

Comment:
I suggest we set a default value for this filed. At least, we use the warning message instead of the error message. 

I was told to minimize the use of sever errors.